src/seoChecks/index.js

```diff
@@ -0,0 +1,33 @@
+const lighthouse = require('lighthouse');
+const chromeLauncher = require('chrome-launcher');
+const fs = require('fs');
+const open = require('open');
+
+function launchChromeAndRunLighthouse(url, opts, config = null) {
+  return chromeLauncher.launch({ chromeFlags: opts.chromeFlags }).then(chrome => {
+    opts.port = chrome.port;
+    return lighthouse(url, opts, config).then(results => {
+      const res = results.report;
+
+      // workaround to save the report
+      fs.writeFile('ceoReport.html', res, err => {
+        if (err) {
+          console.log(err);
+        }
+      });
+      return chrome.kill().then(() => results.lhr);
+    });
+  });
+}
+
+const opts = {
+  output: 'html',
+  chromeFlags: ['--headless'],
+  view: true
+};
+module.exports = seoLink =>
+  launchChromeAndRunLighthouse(seoLink, opts).then(() => {
+    (async () => {
+      await open('./ceoReport.html');
```

Comment:
Excellent!! Can we think something to have multiple files at the same time. So we can see the progression of the app. And also to avoid having the last check erased when a new one is runs.

Maybe adding the time ath the end. For example:
ceoReport-09-08-12-12:02:45.
And we can keep all of these in a folder called ceoReports